extension/httpfs/httpfs-extension.cpp

```diff
@@ -8,7 +8,7 @@
 namespace duckdb {
 
 static void LoadInternal(DatabaseInstance &instance) {
-	S3FileSystem::Verify(); // run some tests to see if all the hashes work out
+	// S3FileSystem::Verify(); // run some tests to see if all the hashes work out
```

Comment:
Forgot to uncomment this?

Reply:
Ah yes, I commented it out because I don't think it's needed anymore, but will check!

Reply:
✔️